man/systemd-cryptenroll.xml

```diff
@@ -251,55 +256,65 @@
           <!-- See: https://github.com/tianocore-docs/edk2-TrustedBootChain/blob/main/4_Other_Trusted_Boot_Chains.md -->
           <!-- See: https://wiki.archlinux.org/title/Trusted_Platform_Module#Accessing_PCR_registers -->
 
-          <tgroup cols='2' align='left' colsep='1' rowsep='1'>
+          <tgroup cols='3' align='left' colsep='1' rowsep='1'>
             <colspec colname="pcr" />
+            <colspec colname="string_alias" />
             <colspec colname="definition" />
 
             <thead>
               <row>
                 <entry>PCR</entry>
+                <entry>alias</entry>
                 <entry>Explanation</entry>
               </row>
             </thead>
 
             <tbody>
               <row>
                 <entry>0</entry>
+                 <entry>corefirmware</entry>
```

Comment:
"core-firmware"

Reply:
> "core-firmware"

This is an awesome awesome community. Thank you!